src/test/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesControllerTest.java

```diff
@@ -34,6 +34,7 @@
 		MISSING_IMAGE_DELIVERIES_DTO = new MissingImageDeliveriesDTO();
 		MISSING_IMAGE_DELIVERIES_DTO.setCompanyName("Test");
 		MISSING_IMAGE_DELIVERIES_DTO.setCompanyNumber("123");
+		MISSING_IMAGE_DELIVERIES_DTO.setFilingHistoryType("TestType");
```

Comment:
This shouldn't have tabs in it, it should use spaces instead. If the entire class is like this, we should raise the formatting as a separate Jira issue. If not, this should be re-formatted.